### ISSUE TITLE
How to debug files_retention?

### ISSUE THREAD

**Issue by helmut72** [nextcloud/files_retention#44]
Hello,

we are using latest Nextcloud 13.0.1 with Files_Retention enabled. Files will be auto-tagged and should be deleted after 30 days, because this local storage is used for temporary files only.

Now I see that even files older than 30 days are not deleted anymore and I don't have any clue why this happens.

How can I debug it? Changing log level doesn't helped that much. I've now changed from unvisible to limited tags and all files are auto-tagged, but the files still won't be deleted.

Thank you!

**Comment by helmut72** [comment]
No one debugging files_retention?

**Comment by nickvergessen** [comment]
You could check if there is a background job which refers to retention and whether that has a last_run, last_checked and reserved_at time

**Comment by hjohanns** [comment]
Hello Helmut, I don't really have a large experience with NC, but I debugged my problems by putting log instructions in the retention job. For instance in 
apps/files_retention/lib/BackgroundJob/RetentionJob.php
I added
`                \OCP\Util::writeLog('cron', 'bgJ Retention removed '.$processed.' files.', \OCP\Util::INFO);
`
at the end of the function run. $processed is a counter into which I accumulate the removed files, just to see the quantities.
